### PR TITLE
Fix Response status property

### PR DESCRIPTION
**Pull request by canac**
The property is `Response.status` not `Response.statusCode`.

### CODE REVIEW

File: test/controllers/api-user.test.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    it('should reject if Okta returns an error', done => {
+      const mockedResponse = {
+        ok: false,
+        status: 500,
+        json: () => Promise.resolve(null)
+      }
+      fetch.mockImplementation(() => Promise.resolve(mockedResponse))
+
+      ApiUserController.lookupUser('bill.bright@cru.org').then(() => {
+        done(new Error('Unexpected success'))
```
Comment (wrandall22) [inline test/controllers/api-user.test.js:56]: I think Jest has a `fail()` function if that would work here.

Comment (canac) [reply test/controllers/api-user.test.js:56]: Can you link to docs for `fail`? I'm having trouble finding it. I found out that I can at least just do `done('Unexpected success')` without passing it an `Error` object. So I think that `done` might double as a `fail` function if passed an argument.

Comment (wrandall22) [reply test/controllers/api-user.test.js:56]: https://jasmine.github.io/api/4.5/global.html#fail

`fail()` is a function of Jasmine, which Jest uses under the hood.

Comment (canac) [reply test/controllers/api-user.test.js:56]: At least for the version of Jest we're using (v27.5.1), Jest uses a fork of Jasmine called jest-jasmine2, and I think they must have removed the fail global.

```js
console.log(Object.keys(globalThis))

[
      'global',         'clearInterval',
      'clearTimeout',   'setInterval',
      'setTimeout',     'Buffer',
      'setImmediate',   'clearImmediate',
      'ArrayBuffer',    'Uint8Array',
      'URL',            'URLSearchParams',
      'TextEncoder',    'TextDecoder',
      'queueMicrotask', 'AbortController',
      'AbortSignal',    'Event',
      'EventTarget',    'performance',
      'atob',           'btoa',
      'process',        'console',
      'afterAll',       'afterEach',
      'beforeAll',      'beforeEach',
      'describe',       'it',
      'test',           'fdescribe',
      'fit',            'xdescribe',
      'xit',            'xtest',
      'expect',         '__core-js_shared__',
      '__fixturesDir'
]
```

Comment (wrandall22) [reply test/controllers/api-user.test.js:56]: Lame

**Review by wrandall22 — APPROVED** [review]

**Comment by wrandall22** [comment]
Looks like lint is failing now.

**Comment by wrandall22** [comment]
It would be nice to squash the last 3 commits together.

**Pull request merged**